exercises/tasks/task_04.py

```diff
@@ -1,8 +1,10 @@
+from exercises.tasks.resources.resource_03_04 import open_kiwi_website_and_accept_cookies
```

Comment:
I'd leave this row out of the exercise task for the attendees to figure out how to import functionality based on their previous, 3rd task.

Reply:
Same applies to the 6th and the following `fill-in-the-blanks` tasks where we introduce a second class to be used (I'd leave it in the task 5 where classes are introduced for the first time, but in the tasks that follow the attendees might be already capable of figuring this out on their own based on their previously finished tasks).

Reply:
Per discussion keeping as the scope of the workshop might be more towards Playwright exercises rather than figuring out how importing in Python works (or having to find the most appropriate function/method to import).